front_end/src/components/landing_page/Landing.js

```diff
@@ -17,15 +37,165 @@ class Landing extends Component {
   };
 
   render() {
-    const { classes } = this.props;
+    let styles = {
+      root: {
+        width: "auto",
+        flexGrow: 1,
+        marginLeft: 150,
+        marginRight: 150
+      },
+      imgSlideShow: {
+        display: "flex",
+        backgroundImage: `url(${SF})`,
+        backgroundSize: "cover",
+        maxWidth: "100%",
+        maxHeight: "100%",
+        height: "70vh",
+        minHeight: 450
+      },
+      searchWidgetBox: {
+        alignSelf: "center",
+        background: "rgba(0,0,0,0.5)",
+        padding: 20,
+        width: "100%"
+      },
+      seperator: {
+        height: 150,
+        width: 300
+      },
+      cityCollage: {
+        width: "auto",
+        marginLeft: 150,
+        marginRight: 150
+      },
+      collageImg: {
+        height: "100%"
+      }
+    };
+
+    function dealStyle(img) {
+      return {
+        display: "flex",
+        alignItems: "center",
+        justifyContent: "center",
+        backgroundImage: `url(${img})`,
+        backgroundSize: "cover",
+        width: 450,
+        height: 300
+      };
+    }
 
-    return <div className="landing" />;
+    return (
+      <div
+        style={{
+          width: "auto",
+          height: "auto",
+          marginLeft: "5%",
+          marginRight: "5%"
+        }}
+      >
+        <div className="shadow" style={styles.imgSlideShow}>
+          <div className="fadeIn" style={styles.searchWidgetBox}>
+            <SearchWidget />
+          </div>
+        </div>
+        <br />
+        <div className="fadeIn">
+          <div>
+            <div style={{ display: "flex", justifyContent: "center" }}>
+              <Typography variant="h3" fontFamily="Roboto">
```

Comment:
can you add the following to index.css:

 \* {
  font-family: Roboto, Helvetica, Arial;
}

and take out the individual fontFamily overrides? It's easier to just have one source for the fontFamily instead of hard coding it into each component.

Reply:
The font family containing those are already added to the Material UI theme which is applied to the entire app. Those individual fontFamily overrides are now removed.

Reply:
can you still add those three lines to index.css?

there might be (most likely) things that are not done with materialUI, and they would then have different fonts :)

Reply:
Done